packages/webdriver/protocol/saucelabs.json

```diff
@@ -108,6 +108,29 @@
       }]
     }
   },
+  "/session/:sessionId/sauce/ondemand/throttle/cpu": {
+    "POST": {
+      "command": "throttleCPU",
+      "description": "You can throttle the CPU in DevTools to understand how your page performs under that constraint.",
+      "ref": "https://wiki.saucelabs.com/display/DOCS/Custom+Sauce+Labs+WebDriver+Extensions+for+Network+and+Log+Commands#CustomSauceLabsWebDriverExtensionsforNetworkandLogCommands-ThrottleNetworkCapabilities",
```

Comment:
the links leads to network throttling instead of cpu

Reply:
The link is still invalid

Reply:
right, sorry, fixing it!

Reply:
updated link